erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -147,6 +148,12 @@ def validate_fixed_asset(self):
 					elif asset.status in ("Scrapped", "Cancelled", "Sold"):
 						frappe.throw(_("Row #{0}: Asset {1} cannot be submitted, it is already {2}").format(d.idx, d.asset, asset.status))
 
+	def validate_item_cost_centers(self):
```

Comment:
1. "Row #{0} in Item list" is not required, just "Row #{0}" is necessary. This is the standard pattern we have used everywhere
2.  Cost center name and company name should be bold

Reply:
@deepeshgarg007 fixed formatting. But I still think "in Item list" should remain? there's a couple of child tables in sales invoice, so it might be hard to understand which row we are talking about.